backend/app/services/employee_emergency_contact_service.py

```
@@ -110,31 +113,31 @@ async def create_emergency_contact(
         await invalidate_cache_prefix("emergency_contact")
-        await invalidate_cache_prefix(f"user:{current_user.user_id}")
-        logger.info(f"Cache invalidated for emergency_contact and user:{current_user.user_id}", extra={"request_id": request_id})
+        await invalidate_cache_prefix(f"user:{user_id}")
+        logger.info(f"Cache invalidated for emergency_contact and user:{user_id}", extra={"request_id": request_id})
 
         # Notify admins
         await _notify_admins_of_contact_change(db, db_contact, current_user, request_id, settings, "created")
 
-        # Log the action
+        # Log the action - use proper serialization
+        new_values_dict = EmployeeEmergencyContactOut.model_validate(db_contact).model_dump(mode='json')
```

Ruff (0.12.2)

Local variable `new_values_dict` is assigned to but never used

Remove assignment to unused variable `new_values_dict`

(F841)